defaults/main.yml

```
@@ -16,6 +16,7 @@ ntfy_identifier: ntfy
 ntfy_base_path: "{{ ntfy_identifier }}"
 ntfy_config_dir_path: "{{ ntfy_base_path }}/config"
 ntfy_data_path: "{{ ntfy_base_path }}/data"
+ntfy_attachment_cache_dir_path: "{{ ntfy_base_path }}/data/cache/attachments"
```

```suggestion
ntfy_attachment_cache_dir_path: "{{ ntfy_data_path }}/cache/attachments"
```

-------

Though.. we can also introduce a separate cache-path as well:

```suggestion
ntfy_cache_path: "{{ ntfy_base_path }}/cache"
ntfy_attachment_cache_dir_path: "{{ ntfy_cache_path }}/attachments"
```

And adjust:

- `tasks/install.yml` to ensure it's created
- `attachment-cache-dir` to match
- the systemd template to ensure `ntfy_cache_path` is mounted under `/cache`